openfl/component/aggregator/aggregator.py

```diff
@@ -82,6 +84,8 @@ def __init__(
         log_memory_usage=False,
         write_logs=False,
         callbacks: Optional[List] = None,
+        persist_checkpoint=True,
```

Comment:
You should also add the `persist_checkpoint` to the default FL plan. 

https://github.com/securefederatedai/openfl/blob/develop/openfl-workspace/workspace/plan/defaults/aggregator.yaml

```
template : openfl.component.Aggregator
settings :
    db_store_rounds : 2
    persist_checkpoint: True
```

This will give users a clue about where to enable / disable this functionality. Would also be useful to add a line to the aggregator plan documentation here: https://github.com/securefederatedai/openfl/blob/6bf229e49678600d516bb0d62d6f2f2f75ff641d/docs/about/features_index/taskrunner.rst#configurable-settings